src/api/docs/content/specs/common.yaml

```diff
@@ -15,6 +15,47 @@ components:
           type: string
           description: Key indicating the status of the request
           example: "success"
+    lists_processed:
```

Comment:
I see what you're doing here, re-using this part for domains/clients/lists/...
However, it' strange to see `example.com` as part of the client example.

![Screenshot at 2023-10-25 22-46-18](https://github.com/pi-hole/FTL/assets/26622301/4ebdd5f4-6f19-4583-ac84-30204758fee2)

Reply:
I split this into individual parts with matching examples and error messages